admin/timescale/create_views.sql

```diff
@@ -1,4 +1 @@
-CREATE MATERIALIZED VIEW listen_count_30day
-       WITH (timescaledb.continuous)
-         AS SELECT time_bucket(bigint '2592000', listened_at) AS listened_at_bucket, user_name, count(listen)
-            FROM listen group by time_bucket(bigint '2592000', listened_at), user_name;
+CREATE MATERIALIZED VIEW listen_count_30day WITH (timescaledb.continuous) AS SELECT time_bucket(bigint '2592000', listened_at) AS listened_at_bucket, user_name, count(listen) FROM listen GROUP BY time_bucket(bigint '2592000', listened_at), user_name;
```

Comment:
Also, We can use `listened_at_bucket` directly in GROUP BY clause. I tried locally and it seemed to work.